core/sequencer/sequencer_api.proto

```diff
@@ -75,8 +70,29 @@ message DefineRevisionsRequest {
 
 // DefineRevisionResponse contains information about freshly defined revisions.
 message DefineRevisionsResponse {
-  // outsanding_revisions a list of all the defined revisions which are not yet applied.
-  repeated int64 outstanding_revisions = 1;
+  // unapplied_begin is the first outstanding revision (inclusive).
+  int64 unapplied_begin = 2;
```

Comment:
I find this a bit harder to grok.  
Was there a particular reason for the change?

Reply:
`outstanding_revisions` always contains a contiguous range of revisions, so this is it just to avoid waste of space. Let me see if I can detach this to a prereq PR.

Reply:
Using revision lists again.